spec/requests/api/service_dialogs_spec.rb

```diff
@@ -62,6 +62,28 @@
 
       expect_result_to_have_only_keys(%w(href id label))
     end
+
+    context 'Delete Service Dialogs' do
+      it 'DELETE /api/service_dialogs/:id' do
+        dialog = FactoryGirl.create(:dialog)
+        api_basic_authorize collection_action_identifier(:service_dialogs, :delete)
+
+        expect do
+          run_delete(service_dialogs_url(dialog.id))
+        end.to change(Dialog, :count).by(-1)
+        expect(response).to have_http_status(:no_content)
+      end
+
+      it 'POST /api/service_dialogs/:id' do
+        dialog = FactoryGirl.create(:dialog)
+        api_basic_authorize collection_action_identifier(:service_dialogs, :delete)
+
+        expect do
+          run_post(service_dialogs_url, 'action' => 'delete', 'resources' => [{ 'id' => dialog.id }])
+        end.to change(Dialog, :count).by(-1)
+        expect(response).to have_http_status(:ok)
+      end
+    end
```

Comment:
Could you add a test that shows deletion of multiple service_dialogs via the "delete" action POST on /api/service_dialogs ? Thanks.